screens/WalletNavigator/screens/WalletOnboarding.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react'
+import { View, Text, TouchableOpacity, ScrollView } from 'react-native'
+import tailwind from 'tailwind-rn'
+import { Ionicons } from '@expo/vector-icons'
+import { translate } from '../../../translations'
+import { PrimaryColor } from '../../../constants/Theme'
+import { useNavigation } from '@react-navigation/native'
+import { useWalletAPI } from '../../../hooks/wallet/WalletAPI'
+import { useDispatch } from 'react-redux'
+
+export function WalletOnboarding (): JSX.Element {
```

Comment:
I would put the main component at the end and all supplementary components before that

Reply:
I think the idea (where I am going with this) is when you open a file you want to read the most important component rather than the child.